dxr/indexers.py

```diff
@@ -73,15 +72,18 @@ class TreeToIndex(PluginConfig):
     and then have it index several files before sending it again.
 
     """
-    def __init__(self, plugin_name, tree):
+    def __init__(self, plugin_name, tree, vcs_cache):
         """
         :arg tree: The configuration of the tree to index: a TreeConfig
+        :arg vcs_cache: A dxr.vcs.VcsCache object. May be None if tree does not
```

Comment:
What we can do here is spell it "A :class:`~dxr.vcs.VcsCache` object", and then Sphinx will link it as a cross reference.